generated/python/gax-google-monitoring-v3/setup.py

```diff
@@ -0,0 +1,40 @@
+"""A setup module for the GAX Monitoring library.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+import sys
+
+install_requires = [
+    'googleapis-common-protos>=1.3.2',
+    'google-gax>=0.12.5, <0.13.0',
+    'grpc-google-monitoring-v3>=0.9.0',
+    'oauth2client>=1.4.11',
+]
+
+setup(
+    name='gax-google-monitoring-v3',
+    version='0.9.0',
```

Comment:
Please re-generate with the latest versions of `artman`, `toolkit`, and `packman`. We're now on the `0.10.x` series with a lot of improvements.